### PR TITLE
Slow blink

### PR DESCRIPTION
**Pull request by ghost**
Studies have shown that 3 seconds is a far better LED delay than 1 second.

http://studies.example.com/optimal-led-delays.html

### CODE REVIEW

File: blink.ino [hunk 1]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  delay(2000);               // wait for a second
```
Comment (ghost) [inline blink.ino:21]: 2000 is best

File: blink.ino [hunk 2]
```diff
@@ -18,7 +18,7 @@ void setup() {
...
   digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
   delay(2000);               // wait for a second
```
Comment (ghost) [inline blink.ino:23]: 3000 is best

File: blink.ino (hunk 1 above)
Comment (ghost) [reply blink.ino:21]: another

File: blink.ino (hunk 2 above)
Comment (ghost) [reply blink.ino:23]: another

File: blink.ino (hunk 1 above)
Comment (fopain) [reply blink.ino:21]: i'm fopain

Comment (fopain) [reply blink.ino:21]: another comment

Comment (fopain) [reply blink.ino:21]: single comment

Comment (fopain) [reply blink.ino:21]: start a review with title and content

Comment (fopain) [reply blink.ino:21]: another comment

**Review by fopain — COMMENTED** [review]
start a review with title and content

啦啦啦

**Comment by ghost** [comment]
comment

**Comment by ghost** [comment]
comment

File: blink.ino [hunk 3]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  delay(2000);               // wait for a second
   digitalWrite(led, LOW);    // turn the LED off by making the voltage LOW
```
Comment (ghost) [inline blink.ino:22]: new

Comment (ghost) [reply blink.ino:22]: i'm contributor

**Review by ghost — COMMENTED** [review]
i'm owner of this pull request

File: blink.ino (hunk 3 above)
Comment (ghost) [reply blink.ino:22]: 啦啦啦

Comment (ghost) [reply blink.ino:22]: 啦啦啦

**Review by ghost — COMMENTED** [review]
啦啦啦

File: blink.ino [hunk 4]
```diff
@@ -18,7 +18,7 @@ void setup() {
 // the loop routine runs over and over again forever:
```
Comment (ghost) [inline blink.ino:18]: 新的评论，不是开启审查

**Comment by fopain** [comment]
done

**Pull request merged**